README.md

```diff
@@ -38,11 +40,15 @@ Red Hat Dependency Analytics only accesses your manifest files to analyze your a
 
 **Prerequisites**
 
-- For Maven projects, analyzing a `pom.xml` file, you must have the `mvn` binary in your system’s `PATH` environment.
-- For Node projects, analyzing a `package.json` file, you must have the `npm` binary in your system’s `PATH` environment.
-- For Golang projects, analyzing a `go.mod` file, you must have the `go` binary in your system’s `PATH` environment.
-- For Python projects, analyzing a `requirements.txt` file, you must have the `python3/pip3` or `python/pip` binaries in your system’s `PATH` environment.
-- For Gradle projects, analyzing a `build.gradle` file, you must have the `gradle` binary in your system's `PATH` environment.
+- For Maven projects, analyzing a `pom.xml` file, you must have the `mvn` binary in your system's `PATH` environment.
+- For NPM projects, analyzing a `package.json` file, you must have the `npm` binary in your system's `PATH` environment.
+- For PNPM projects, analyzing a `package.json` file, you must have the `pnpm` binary in your system's `PATH` environment.
+- For Yarn projects, analyzing a `package.json` file, you must have the `yarn` binary in your system's `PATH` environment.
+  <br >**Note:** For NPM, PNPM, and Yarn projects, you can use [fnm](https://github.com/Schniz/fnm) or [nvm](https://github.com/nvm-sh/nvm) for Node.js version management. The extension will automatically detect and use the binary path from the `FNM_DIR` or `NVM_DIR` environment variables.
+- For Golang projects, analyzing a `go.mod` file, you must have the `go` binary in your system's `PATH` environment.
+- For Python projects, analyzing a `requirements.txt` file, you must have the `python3/pip3` or `python/pip` binaries in your system's `PATH` environment.
+- For Gradle Groovy DSL projects, analyzing a `build.gradle` file, you must have the `gradle` binary in your system's `PATH` environment.
+- For Gradle Kotlin DSL projects, analyzing a `build.gradle` file, you must have the `gradle` binary in your system's `PATH` environment.
```

Comment:
This isnt technically a requirement, given that users can configure the path e.g. through `redHatDependencyAnalytics.mvn.executable.path`

Reply:
I have updated the description to mention this as well